docs/.vitepress/Footer.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <footer class="footer">
+    <div class="footer-column logo-column">
+      <img src="/warp-logo.svg" alt="Warp Logo" width="122" />
+    </div>
+    <div class="footer-column">
+      <h3 class="footer-column-title">Collaborate</h3>
+      <ul>
+        <li class="footer-link">
+          <a :href="`${baseUrl}collaborate/contribute/#adding-a-new-component`">Request new component</a>
+        </li>
+        <li class="footer-link">
+          <a :href="`${baseUrl}collaborate/contribute/#requesting-changes-to-an-existing-component`">Request changes to a component</a>
+        </li>
+      </ul>
+    </div>
+    <div class="footer-column">
+      <h3 class="footer-column-title">Help</h3>
+      <ul>
+        <li class="footer-link">
+          <a :href="`${baseUrl}help/support`">Support</a>
+        </li>
+        <li class="footer-link">
+          <a :href="`${baseUrl}help/report-bugs`">Report bugs/errors</a>
+        </li>
+      </ul>
+    </div>
+  </footer>
+</template>
+
+<script setup>
+const baseUrl = import.meta.env.BASE_URL
+</script>
+
+<style scoped>
```

Comment:
Should we get rid of the styles and use warp classes as much as possible?

Reply:
Sure I can change it! :) I just added the styles block for now because I wasn't supposed to add styling yet. But makes sense to prepare it already using warp classes instead 👍